services/test/services/errortelemetry.test.js

```diff
@@ -1,31 +1,63 @@
-const assert = require('assert');
+const fs             = require('fs');
+const assert         = require('assert');
+
 const app = require('../../src/app');
 
+const errorTelemetryService = 'telemetry/error';
```

Comment:
This is also lacking a leading slash, which makes me wonder if these tests are working properly or if Feathers is being smart enough.

Regardless, I think it's important that our references to services remain consistent